frontend/src/app/main/ui/dashboard/team.cljs

```diff
@@ -571,6 +577,229 @@
+                                 "")) ;; TODO Add more error codes when back defines them
+                          message)]
+            (rx/of (msg/error message))))
+
+        on-create-submit
+        (fn []
+          (let [mdata   {:on-success #(on-success (tr "dashboard.webhooks.create.success"))
+                         :on-error   (partial on-error (tr "dashboard.webhooks.create.error"))}
+                webhook {:uri        (get-in @form [:clean-data :uri])
+                         :mtype      (get-in @form [:clean-data :mtype])
+                         :is-active  (get-in @form [:clean-data :is-active])}]
+            (st/emit! (dd/create-team-webhook (with-meta webhook mdata)))))
+
+        on-update-submit
+        (fn []
+          (let [mdata   {:on-success #(on-success (tr "dashboard.webhooks.update.success"))
+                         :on-error   (partial on-error (tr "dashboard.webhooks.update.error"))}
+                webhook (get @form :clean-data)]
+            (st/emit! (dd/update-team-webhook (with-meta webhook mdata)))))
+
+        on-submit
+        #(let [data (:clean-data @form)]
+           (if (:id data)
+             (on-update-submit)
+             (on-create-submit)))]
+
+    [:div.modal-overlay
+     [:div.modal-container.webhooks-modal
+      [:& fm/form {:form form :on-submit on-submit}
+
+       [:div.modal-header
+        [:div.modal-header-title
+         (if webhook
+           [:h2 (tr "modals.edit-webhook")]
+           [:h2 (tr "modals.create-webhook")])]
+
+        [:div.modal-close-button
+         {:on-click #(st/emit! (modal/hide))} i/close]]
+
+       [:div.modal-content.generic-form
+        [:div.fields-container
+         [:div.fields-row
+          [:& fm/input {:type "text"
+                        :auto-focus? true
+                        :form form
+                        :name :uri
+                        :label (tr "modals.create-webhook.url.label")
+                        :placeholder (tr "modals.create-webhook.url.placeholder")}]]
+
+         [:div.fields-row
+          [:& fm/select {:options mtypes
+                         :label (tr "dashboard.webhooks.content-type")
+                         :default "application/json"
+                         :name :mtype}]]]
+        [:div.fields-row
+         [:div.input-checkbox.check-primary
+          [:& fm/input {:type "checkbox"
+                        :form form
+                        :name :is-active
+                        :label (tr "dashboard.webhooks.active")}]
+          ]
+         [:div.explain (tr "dashboard.webhooks.active.explain")]]]
+
+
+
+       [:div.modal-footer
+        [:div.action-buttons
+         [:input.btn-gray.btn-large
+          {:type "button"
+           :value (tr "labels.cancel")
+           :on-click #(modal/hide!)}]
+         [:& fm/submit-button
+          {:label (if webhook
+                    (tr "modals.edit-webhook")
```

Comment:
I think, you miss the 3rd level on message indicating how it is:

- `modals.edit-webhook.submit-label`
- `modals.create-webhook.submit-label`

Reply:
Fixed